### PR TITLE
feat: add a help button throughout many parts of hub

### PR DESCRIPTION
**Pull request by Sparkier**
# Description

fix ZEN-343

https://github.com/zeno-ml/zeno-hub/assets/5690524/41823a49-29bd-4bea-b14b-58cbdc24bbcf

### CODE REVIEW

**Comment by cabreraalex** [comment]
This is EPIC

File: frontend/src/lib/components/general/Help.svelte [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	let showOptions = false;
+</script>
+
+<div
+	class="relative z-20 flex flex-col items-end"
+	transition:fade={{ delay: 1000, duration: 1000 }}
+>
+	<IconButton
+		class="border border-primary-light opacity-25 transition hover:opacity-100"
+		on:click={() => (showOptions = !showOptions)}
```
Comment (cabreraalex) [inline frontend/src/lib/components/general/Help.svelte:21]:
```suggestion
		class="border-primary-lighter rounded-full border bg-primary-mid opacity-50 transition hover:opacity-100"
```

**Comment by cabreraalex** [comment]
I like it. Two opinions:
* Can we make it a bit darker? Really hard to find right now if you don't know where to look. Left a suggestion
* Another option is putting it next to the account button? Would add less clutter and easier to find? but not as distinctive. can go either way.

**Comment by cabreraalex** [comment]
If we leave it on the bottom right, we should have the margin line up with the header on the home page and reports (especially x axis) it's bothering me a bit lol

**Comment by Sparkier** [comment]
image

**Comment by Sparkier** [comment]
I think I like this even better:
image

**Comment by Sparkier** [comment]
I prefer the "always tucked to the right edge of the browser one.

I tried next to the account button already yesterday as well but looked super ugly.

**Comment by cabreraalex** [comment]
Love the circle one

**Review by cabreraalex — APPROVED** [review]
LGTM - maybe we add it to the account page?

**Pull request merged**